### ISSUE TITLE
Unable to Claim JavaScript Certificate!

### ISSUE THREAD

**Issue by ktsmpng** [freeCodeCamp/freeCodeCamp#34412]
**Describe the bug**
When I go to view the claimed certificate it displays a pop up saying that I have not accepted the Academic Honest Policy but I have already accepted it in the settings. I have tried using different browsers, signing out logging in and refreshing cache but none of this has worked. I viewed another issue that was similar to this which was resolved. The user had two accounts in the same email. perhaps this is the issue for me? I had my account before the new update.

**Screenshots**
![image](https://user-images.githubusercontent.com/23035767/48864141-92acbd80-edc3-11e8-8376-924e9035c154.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Home 
 - Browser: Chrome, Opera
 - Version: 70.0.3538.102

**Comment by thecodingaviator** [comment]
Try a hard reload once again. Will have to check why this is happening

**Comment by thecodingaviator** [comment]
Can you open the browser console/developer tools console and check if any warning/errors are there?

**Comment by ktsmpng** [comment]
![image](https://user-images.githubusercontent.com/23035767/48906221-b28fc000-ee5b-11e8-8c19-6fb5bd7693e9.png)

When I click the view certificate button it refreshes the page then displays this. I did this after hard resetting cache.

**Comment by thecodingaviator** [comment]
Can you please also attach a screenshot of the Academic Honesty pledge module where it says you've accepted it?

**Comment by ktsmpng** [comment]
![image](https://user-images.githubusercontent.com/23035767/48909000-f1c20f00-ee63-11e8-951b-8215fd291ba6.png)

**Comment by thecodingaviator** [comment]
You said something about 2 accounts on the same email, could you please elaborate, have you deleted the other one?

**Comment by ktsmpng** [comment]
I am not sure if I have two accounts but I read on a different issue in regards to FCC that another user had two accounts under same email and issue was resolved after. I can't find the issue again on github. This will have to be checked by someone who has access to the database

**Comment by ktsmpng** [comment]
I signed in with my google account and the account I noticed were not synced with my github one. I re-added the projects on the other account accepted policy and the certification was able to show. There is still an issue in the db but the root cause has been found. Is there someone that can help me with it or can I just delete the google account I dont want to lose my progress but I dont want to keep updating both.
